Build failing on Windows

```
1>------ Build started: Project: game-shared, Configuration: Release x64 ------
2>------ Build started: Project: engine-shared, Configuration: Release x64 ------
1>Generating src/game/generated/git_revision.cpp
2>'C:\Windows\System32\python.exe' is not recognized as an internal or external command,
2>operable program or batch file.
2>Generating src/game/generated/protocol.cpp
1>'C:\Windows\System32\python.exe' is not recognized as an internal or external command,
1>operable program or batch file.
1>C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(209,5): error MSB6006: "cmd.exe" exited with code 9009.
1>Done building project "game-shared.vcxproj" -- FAILED.
2>C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(209,5): error MSB6006: "cmd.exe" exited with code 9009.
2>Done building project "engine-shared.vcxproj" -- FAILED.
3>------ Build started: Project: DDNet, Configuration: Release x64 ------
3>'C:\Windows\System32\python.exe' is not recognized as an internal or external command,
3>operable program or batch file.
3>Generating src/game/generated/client_data.cpp
3>C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(209,5): error MSB6006: "cmd.exe" exited with code 9009.
3>Done building project "DDNet.vcxproj" -- FAILED.
========== Build: 0 succeeded, 3 failed, 7 up-to-date, 0 skipped ==========
```

```
CMake Warning at CMakeLists.txt:292 (find_package):
  By not providing "FindPython.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Python", but
  CMake did not find one.

  Could not find a package configuration file provided by "Python" with any
  of the following names:

    PythonConfig.cmake
    python-config.cmake

  Add the installation prefix of "Python" to CMAKE_PREFIX_PATH or set
  "Python_DIR" to a directory containing one of the above files.  If "Python"
  provides a separate development package or SDK, be sure it has been
  installed.


Found PythonInterp: C:/Windows/System32/python.exe (found version "3.6.6")
```

Probably caused by #1344. @yangfl Have you tested on Windows? Does anyone else have a Windows to debug this?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -289,12 +289,7 @@ find_package(Ogg)
 find_package(Opus)
 find_package(Opusfile)
 find_package(Pnglite)
-find_package(Python COMPONENTS Interpreter)
-if(NOT PYTHON_FOUND)
-  find_package(PythonInterp)
-  set(Python_EXECUTABLE "${PYTHON_EXECUTABLE}")
-  set(PYTHON_FOUND "${PYTHONINTERP_FOUND}")
-endif()
+find_package(PythonInterp)
 find_package(SDL2)
 find_package(Threads)
 find_package(Wavpack)
@@ -358,7 +353,7 @@ show_dependency_status("OpenSSL Crypto" CRYPTO)
 show_dependency_status("Opus" OPUS)
 show_dependency_status("Opusfile" OPUSFILE)
 show_dependency_status("Pnglite" PNGLITE)
-show_dependency_status("Python" PYTHON)
+show_dependency_status("PythonInterp" PYTHONINTERP)
 show_dependency_status("SDL2" SDL2)
 show_dependency_status("Wavpack" WAVPACK)
 show_dependency_status("Zlib" ZLIB)
@@ -369,7 +364,7 @@ endif()
 if(NOT(CURL_FOUND))
   message(SEND_ERROR "You must install Curl to compile the DDNet")
 endif()
-if(NOT(PYTHON_FOUND))
+if(NOT(PYTHONINTERP_FOUND))
   message(SEND_ERROR "You must install Python to compile DDNet")
 endif()
 
@@ -522,7 +517,7 @@ file(COPY ${COPY_FILES} DESTINATION .)
 
 function(generate_source output_file script_parameter)
   add_custom_command(OUTPUT ${output_file}
-    COMMAND ${Python_EXECUTABLE} datasrc/compile.py ${script_parameter}
+    COMMAND ${PYTHON_EXECUTABLE} datasrc/compile.py ${script_parameter}
       > "${PROJECT_BINARY_DIR}/${output_file}"
     DEPENDS
       datasrc/compile.py
@@ -547,7 +542,7 @@ if(NOT PROJECT_GIT_DIR_ERROR)
   )
 endif()
 add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/src/game/generated/git_revision.cpp
-  COMMAND ${Python_EXECUTABLE}
+  COMMAND ${PYTHON_EXECUTABLE}
     scripts/git_revision.py
     > ${PROJECT_BINARY_DIR}/src/game/generated/git_revision.cpp
   WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
@@ -1405,7 +1400,7 @@ if(CLIENT AND (DMGTOOLS_FOUND OR HDIUTIL))
     COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:${TARGET_SERVER}> $<TARGET_FILE:${TARGET_SERVER_LAUNCHER}> ${DMG_TMPDIR}/${TARGET_SERVER}.app/Contents/MacOS/
 
     # DMG
-    COMMAND ${Python_EXECUTABLE} ${PROJECT_SOURCE_DIR}/scripts/dmg.py create ${DMG_PARAMS} ${CPACK_PACKAGE_FILE_NAME}.dmg ${CPACK_PACKAGE_FILE_NAME} ${DMG_TMPDIR}
+    COMMAND ${PYTHON_EXECUTABLE} ${PROJECT_SOURCE_DIR}/scripts/dmg.py create ${DMG_PARAMS} ${CPACK_PACKAGE_FILE_NAME}.dmg ${CPACK_PACKAGE_FILE_NAME} ${DMG_TMPDIR}
 
     DEPENDS
       ${TARGET_CLIENT}

